src/coreclr/src/jit/compiler.h

```diff
@@ -3614,7 +3614,7 @@ class Compiler
                           CORINFO_CLASS_HANDLE    clsHnd,
                           CORINFO_METHOD_HANDLE   method,
                           CORINFO_SIG_INFO*       sig,
-                          unsigned                methodFlags,
+                          unsigned&               methodFlags,
```

Comment:
Had to drop `CORINFO_FLG_VIRTUAL` from methodflags due to this assert: https://github.com/dotnet/runtime/blob/76b53dbf84374b8be8091f218d2c5841807e9de5/src/coreclr/src/jit/importer.cpp#L7573

Reply:
I think it would be better to delete the assert than trying to patch `methodFlags` to make it happy.